src/core/midi/MidiSndio.cpp

```diff
@@ -27,17 +27,16 @@
 
 #ifdef LMMS_HAVE_SNDIO
 
-#include <QLabel>
-#include <QLineEdit>
 
 #ifdef LMMS_HAVE_STDLIB_H
-#include <stdlib.h>
+#include <cstdlib>
 #endif
```

Comment:
Should the CMake header detection be updated to check for the new header?

Reply:
Can we even remove some checks? According to https://en.cppreference.com/w/cpp/freestanding, it looks `<cstdlib>` and `<cstdint>` can be included without any checks.

Reply:
Agreed to completely remove, especially since we often use that header without that macro and it still works.

Reply:
I removed those 2 now completely from LMMS. From your link, there seem to be no more such headers, so that should be enough.